ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/LoggingInterceptor.java

```diff
@@ -64,6 +65,9 @@ public Object processInvocation(InterceptorContext interceptorContext) throws Ex
         try {
             // we just pass on the control and do our work only when an exception occurs
             return interceptorContext.proceed();
+        } catch ( EJBComponentUnavailableException ex) {
+            EjbLogger.EJB3_INVOCATION_LOGGER.warn(ex);
```

Comment:
IMO this should invoke a new method added to EjbLogger that takes the component name, the invoked method and the value of ex.toString(). I believe the way it is now there will just be this in the logs, perhaps repeatedly:

```
WARN  [org.jboss.as.ejb3.invocation] (EJB default - 7) WFLYEJB0467: The request was rejected as the container is suspended
```

That meets the letter of the law in section 14.3 of the EJB3.1 spec, but it's not particularly useful as there's no context in the message.